### DISCUSSION TITLE
Flood light cam

### DISCUSSION THREAD

**Discussion by airbornetrooper82573** [mrlt8/docker-wyze-bridge#245] [General]
Has anyone got their flood light cam to work? It's essentially a v3 + flood light.

**Comment by airbornetrooper82573** [comment]
Latest update allowed my flood light to start streaming so you can add that on the confirmed list!

**Reply by jsiemonski** [reply]
Did you have to do anything special to it? I just got one and I can't seem to get it to work. It initially shows up, but it seems to hose the connection, and I have to disable this add-on to make it work in the Wyze app at all.

**Reply by airbornetrooper82573** [reply]
At the time it worked. I am on the latest firmware beta and it stopped working recently

**Reply by jsiemonski** [reply]
I am on 4.36.8.15.
